### DISCUSSION TITLE
How to avoid outputting blue debugging commands on the terminal for a more concise display.

### DISCUSSION THREAD

**Discussion by pcsensor** [microsoft/vscode-discussions#963] [Extension Development QnA]
![1701914020566](https://github.com/microsoft/vscode-discussions/assets/99247580/2b25548e-35cc-482c-9bb8-f0eb8c2a7e8a)

**Comment by olguzzar** [comment]
Hi there, check out this [Stack Overflow question](https://stackoverflow.com/questions/62345409/vs-code-shows-print-output-in-terminal-instead-of-output) that is similar to yours. As a note, this forum is specifically for extension developers, so I will be closing this discussion. Thanks!
